app-accessibility/brltty/brltty-6.1.ebuild

```diff
@@ -46,6 +48,7 @@ DEPEND="
 		app-accessibility/flite
 		app-accessibility/speech-dispatcher
 	)
+	sys-apps/dbus
```

Comment:
[We](https://devmanual.gentoo.org/general-concepts/dependencies/) do all unconditional then conditional, so it was right before